app/src/main/java/com/github/sdpsharelook/Section/Section.kt

```diff
@@ -9,7 +10,13 @@ val SECTION_ID = "sectionExtra"
 class Section(
     var title: String,
     var flag: Int,
-    val databaseRepo: RTDBWordListRepository,
+    val databaseRepo: RTDBWordSectionRepository,
     val sectionRepo: String,
     val id: Int? = sectionList.size
```

Comment:
SectionList.size shouldn't be used for ids, rename variable if needed to avoid confusion.